common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -190,7 +190,10 @@ private static async Task RunTestsAsync(IPerfTest[] tests, PerfOptions options,
             var latency = warmup ? false : options.Latency;
 
             _completedOperations = new int[options.Parallel];
-            _lastCompletionTimes = new TimeSpan[options.Parallel];
+
+            // Initialize _lastCompletionTimes with non-zero values to avoid divide-by-zero for threads
+            // with no completed operations.
+            _lastCompletionTimes = Enumerable.Repeat(TimeSpan.MinValue, options.Parallel).ToArray();
```

Comment:
So what are we going to get now by dividing-by-min-value? would it be a negative value? or would it be 0?

Reply:
The default `_lastCompletionTime` is only used when the numerator is 0, so the value doesn't matter as long as it's non-zero.  We just want to avoid divide-by-zero when the numerator (number of completed operations) is zero, and there is also "no last completion time" since nothing has been completed.  This is actually a fairly important but subtle issue, so I will put most of my comments in https://github.com/Azure/azure-sdk-for-net/issues/18510.